tests/Recipe-test.js

```diff
@@ -74,4 +75,19 @@ describe('Recipe', () => {
   it('should return instructions', () => {
     expect(recipe1.getInstructions()).to.be.equal(prototypeRecipes[0].instructions);
   });
+
+  it('should return an array of the ingredients with prices', () => {
+    const estimatedIngredients = [
+      { id: 1, name: 'pumpkin', estimatedCostInCents: 142 },
+      { id: 2, name: 'egg nog', estimatedCostInCents: 582 },
+      { id: 3, name: 'cinnamon', estimatedCostInCents: 472 },
+      { id: 4, name: 'nutmeg', estimatedCostInCents: 582 }
+    ]
+
+    expect(recipe1.getIngredientWithPrices()).to.deep.equal(estimatedIngredients);
```

Comment:
This took me a bit to understand, but I get it now. There are SO many ways to solve this project. It's really cool!!

Reply:
Because of personal life stuff, I am going ahead and merging this branch. We can come back and refactor! :)